staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -182,6 +188,12 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"it out. This is the default request timeout for requests but may be overridden by flags such as "+
 		"--min-request-timeout for specific types of requests.")
 
+	fs.Float64Var(&s.GoawayChance, "goaway-change", s.GoawayChance, ""+
+		"An optional field indicating the probability to send GOAWAY to HTTP2 client. When Client received"+
+		"GOAWAY, the in-flight requests will not be influenced and new requests will use"+
+		"a new TCP connection to trigger re-balancing to another server behind the load balance."+
```

Comment:
Rewrite this:
`To prevent HTTP/2 clients from getting stuck on a single apiserver, randomly close a connection (GOAWAY). The client's other in-flight requests won't be affected, and the client will reconnect, likely landing on a different apiserver after going through the load balancer again. This argument sets the fraction of requests that will be sent a GOAWAY. Clusters with single apiservers, or which don't use a load balancer, should NOT enable this. Min is 0 (off), Max is .02 (1/50 requests); .001 (1/1000) is a recommended starting point.`

Reply:
@deads2k for fact checking.

Reply:
> @deads2k for fact checking.

Looks accurate to me.  I think we can reserve the right to change the default based on experience, but 1/1000 seems like a good starting point for the clusters that will care about this.

Reply:
I agreed 1/1000 is a good start point too. From the perspective of the client, it set up a new connection after 1000 times visit API Server.  It not only takes care of the performance of client reuse connection, but also ensures that the load of API Server(s) is balanced timely.